README.md

```diff
@@ -772,7 +771,7 @@ Any <kbd>Ctrl</kbd> shortcut is forwarded to the device. In particular:
 This typically works as you expect.
 
 The actual behavior depends on the active application though. For example,
-_Termux_ sends SIGINT on <kbd>Ctrl</kbd>+<kbd>c</kbd> instead, and _K-9 Mail_
+`Termux` sends SIGINT on <kbd>Ctrl</kbd>+<kbd>c</kbd> instead, and `K-9 Mail`
```

Comment:
(I prefer italic for program names.)

Reply:
Sounds good.  What about for `scrpy` and `adb`?

Consistency will help add clarity.  What do you think?

Reply:
I would say, as a general rule:
 - _italic_ when it's the program name ("the _adb_ connection")
 - `backquotes` when it's the command ("run `adb`")

(Sometimes, the distinction is not clear.)